.storybook/stories/KvMap.stories.js

```diff
@@ -5,19 +5,32 @@ export default {
 	component: KvMap,
 	args: {
 		autoZoomDelay: 1000,
+		height: null,
 		initialZoom: null,
 		lat: 37.700091,
 		long: -123.013243,
 		mapId: 0,
 		useLeaflet: false,
+		width: null,
```

Comment:
Added the width and height props here and in the template to fix up the stories!